website/docs/r/license.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: "vsphere"
+page_title: "VMware vSphere: vsphere_license"
+sidebar_current: "docs-vsphere-resource-license"
+description: |-
+  Provides a VMware vSphere license resource. This can be used to add and remove license keys.
+---
+
+# vsphere\_license
+
+Provides a VMware vSphere license resource. This can be used to add and remove license keys.
+
+## Example Usage
+
+```hcl
+resource "vsphere_license" "licenseKey" {
+  license_key = "452CQ-2EK54-K8742-00000-00000"
+
+  label {
+    key = "VpxClientLicenseLabel"
+    value ="Hello World"
+  }
+  label {
+    key = "Workflow"
+    value ="Hello World"
+  }
+  
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `license_key` - (Required) The key of the license which is needed to be added to vpshere
+* `label` - (Optional) The key value pair of labels that has to be attached to the license key. One key can have multiple labels.
```

Comment:
I think this could probably do with more detail. Something like:

>  * `label` - (Optional) A set of labels to apply to the license key. May be specified multiple times to supply multiple label keys.
> 
> The values for `labels` are:
>  * `key` - (Required) The key for the label.
>  * `values` - (Required) A list of string values to assign to the label key.
> 

The latter note on `values` also takes into consideration my notes on the `labels` attribute in the schema itself - so check there for more details.